web/accounts/tests/test_auth_views.py

```
@@ -0,0 +1,53 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.fixture
+def existing_user():
+    User = get_user_model()
+    return User.objects.create_user(email="alice@example.com", password="pw12345678")
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "password"

(S106)